src/zip.c

```diff
@@ -160,7 +160,7 @@ struct zip_t *zip_open(const char *zipname, int level, char mode) {
   case 'a':
     if (!mz_zip_reader_init_file(
             &(zip->archive), zipname,
-            level | MZ_ZIP_FLAG_DO_NOT_SORT_CENTRAL_DIRECTORY)) {
+            (mz_uint)level | MZ_ZIP_FLAG_DO_NOT_SORT_CENTRAL_DIRECTORY)) {
```

Comment:
I think here would be better to use: `zip->level`

Reply:
Agree